src/Discord/Messaging.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Deployer\Discord;
```

Comment:
Can it be done without using extra classes?

Reply:
Hey @antonmedv! I think yes, but why?
I mean, I think that separating things could be a good way to organise the code.
What do you think?
🙂

Reply:
Yes, but not v6 have some problems with autoloading and phars. I'm going to solve this in future version bout to work correctly on all version of deployer now must not include composer autoload.

Reply:
@antonmedv got you! Thanks!

I'll update my code then. :)